.all-contributorsrc

```diff
@@ -655,6 +655,16 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "jcarral",
+      "name": "Joseba Carral",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/13391685?v=4",
+      "profile": "http://codevs.es",
+      "contributions": [
+        "code",
+        "plugin"
+      ]
```

Comment:
Use `translation` instead of `code` and `plugin`

Reply:
Yes, sorry, i did it quickly. You're right it's translation 😅